modules/gitops-release-notes-1-14-1.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-1_{context}"]
+= Release notes for {gitops-title} 1.14.1
+
+{gitops-title} 1.14.1 is now available on {OCP} 4.13, 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-14.1_{context}"]
+== Errata updates
+
+[id="RHEA-XXXX:NNNN-gitops-1-14-1-security-update-advisory_{context}"]
+=== RHEA-XXXX:NNNN - {gitops-title} 1.14.1 security update advisory
+
+Issued: 2024-XX-YY
+
+The list of security fixes that are included in this release are documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-1_{context}"]
+== Fixed issues
+
+* Before this update, resource limits set on the operator container led to functional issues on clusters containing a large number of `Secrets` and `ConfigMaps`. As a result, the operator container exceeded its allocated memory limits. With this update, the resource limits set on the operator container are removed to enhance functionality. link:https://issues.redhat.com/browse/GITOPS-5665[GITOPS-5665]
```

Comment:
The text doesn't fully capture the intended change. Could you please revise it to align more closely with the RN text from JIRA?

Reply:
I have made a change that hopefully addresses this comment, PTAL.